### DISCUSSION TITLE
Violation of Twitter's guidelines on logo usage

### DISCUSSION THREAD

**Discussion by zignis** [tabler/tabler-icons#631] [General]
As per Twitter's brand guidelines (available at https://about.twitter.com/content/dam/about-twitter/en/brand-toolkit/downloads/twitter-external-brand-guidelines-01272021.pdf), it is prohibited to modify their logo, including the outlined version available in Tabler Icons, as well as every other social icon in this repository. What are your thoughts on this, and is it possible to use these icons for commercial projects while still adhering to Twitter's guidelines?

And thank you very much for providing these fantastic icons!

**Comment by codecalm** [comment]
The Twitter icon was added as a request from users that they wanted icons associated with brands. It was made in the same style as all other icons: 24x24px and 2px stroke. When designing, I try to replicate the icons of well-known brands as much as possible. Unfortunately, it is not always possible to do this 100%.

**Reply by zignis** [reply]
Understandably, the social icons were designed based on user requests, so they should be used with extreme care. Personally, I would recommend using the social icons provided by the brand itself to avoid any potential issues in the future. Thank you for the clarification!

**Comment by daxpandhi** [comment]
I can say this from inside experience: Most companies have no problem with their logo being modified for icon usage. The logo guidelines tend to be for larger scale and more prominent use. Especially when someone tries to misrepresent the brand.

**Reply by zignis** [reply]
Looks reasonable. I will try to contact their branding team, just to be safe.
